README.md

```diff
@@ -1,55 +1,56 @@
 # Autoware (Architecture Proposal)
 
-meta-repository for Autoware architecture proposal version
-
 ![autoware](https://user-images.githubusercontent.com/8327598/69472442-cca50b00-0ded-11ea-9da0-9e2302aa1061.png)
 
-# What's this
-
-This is the source code of the feasibility study for Autoware architecture proposal.
-
-> **WARNING**: This source is solely for demonstrating an architecture proposal. It should not be used to drive cars. 
+A meta-repository for the Autoware architecture proposal feasibility study created by Tier IV. For more details about the architecture itself, please read this [overview](/design/Overview.md).
 
-> **NOTE**: The features in [AutowareArchitectureProposal.iv](https://github.com/tier4/AutowareArchitectureProposal.iv) will be merged into Autoware.Auto.
+> **WARNING**: All source code relating to this meta-repository is solely intended to demonstrate a potential new architecture for Autoware, and should not be used to autonomously drive a real car!
+> 
+> **NOTE**: The features within the [AutowareArchitectureProposal.iv repository](https://github.com/tier4/AutowareArchitectureProposal.iv) are planned to be merged into [Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto).

```

Comment:
I think it should be made clear that not _all_ features will be merged. As AutowareAuto has its own scope and ODD which it needs to achieve, not all the features here will be required. It would be nice to link to an issue that tracks the porting but I don't believe we have one on the AutowareAuto side, at least not a single issue.